discovery.rst

```diff
@@ -84,23 +84,52 @@ Read more about setting up Puli in their `official documentation`_.
 Common Errors
 -------------
 
+Could not find resource using any discovery strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you get an error saying "*Could not find resource using any discovery strategy.*" it means that all the
+discovery [#strategies]_ have failed. The cause of this is probably because you have not installed message factories
```

Comment:
maybe the "Too many autonumbered footnote references" error is because of `[#strategies]_` - i am not sure what this would be but our other links use the format ``:ref:`text <targetname>```